docs/UserGuide.md

```diff
@@ -155,10 +155,10 @@ Format:  `add n/NAME d/DESCRIPTION dl/DEADLINE p/PRIORITY [t/TAG]…`
 
 Example: `add n/CS2103T tp meeting d/read the weekly tasks before the meeting dl/2022-03-27 p/medium t/CS2103T t/meeting`
 
-The constraints on the inputs can be found [here](#6-appendix-constraints-on-inputs).
+![Add a task](images/Ui-AddTask.png)
+Figure 3.1: Example of Harmonia after adding a task
```

Comment:
Should the figure captions be italicized for consistency with the other section?

Reply:
Thanks for pointing that out! Have updated it in the latest commit!